examples/wmem-gb-alert-report.json

```diff
@@ -0,0 +1,16 @@
+{
+  "specversion": "1.0",
+  "type": "int.wmo.wis.wme.event.report.alert",
+  "source": "fr-meteofrance-global-broker",
+  "subject": "us-noaa-synoptic",
+  "id": "7bdd817e-5f80-4d58-ab6e-fc5beaa9b99b",
+  "time": "2025-11-10T15:24:56.086Z",
+  "datacontenttype": "application/json",
+  "dataschema": "https://raw.githubusercontent.com/wmo-im/wis2-monitoring-events/refs/heads/main/schemas/wnm-report-bundled.json",
+  "data": {
+    "type": "wmem-report:notice",
```

Comment:
Notice or report ?
And if it is a report (which, I think is better), as there is no alert (just a metric) then we cannot follow the rule 
The data.type property of a WMEM Report for an alert SHALL be based on [WIS2 Metric Hierarchy](https://wmo-im.github.io/wis2-monitoring-events/standard/wis2-monitoring-events-DRAFT.html#wis2-metric-hierarchy) alert names OR have the fixed value of wmem:report:notice.